.github/workflows/katib_kind_test.yaml

```diff
@@ -35,6 +35,6 @@ jobs:
 
     - name: Create katib experiment
       run: |
-        kubectl apply -f tests/gh-actions/katib_test.yaml
+        kubectl apply -f tests/gh-actions/kf-objects/katib_test.yaml
```

Comment:
Looks like the resources are created under `kubeflow-user` namespace which is not created before. In addition, it looks for a label to be enabled to create the experiment.

Running the follow commands before applying this yaml file made it work for me. Otherwise, it will throw an error for missing namespace

```
kubectl create namespace kubeflow-user
kubectl label namespace kubeflow-user katib.kubeflow.org/metrics-collector-injection=enabled
```

Reply:
Good catch! I'll send a follow up PR asap to fix this